fjord/base/browsers.py

```diff
@@ -98,11 +98,14 @@ def parse_ua(ua):
             browser_version = part[1]
         elif 'Iceweasel' in part:
             browser = 'Iceweasel'
+        elif 'FxiOS' in part:
+            browser = 'Firefox for iOS'
+            browser_version = part[1]
 
     platform = platform_parts.pop(0)
     platform_version = UNKNOWN
 
-    while platform in ['X11', 'Ubuntu', 'U']:
+    while platform in ['X11', 'Ubuntu', 'U', 'iPad', 'iPhone']:
```

Comment:
Yeah... I think I'm not going to worry about that for the rewrite and instead push it off until someone asks for it.